pkg/gui/gui.go

```diff
@@ -135,7 +135,8 @@ type Gui struct {
 	c       *helpers.HelperCommon
 	helpers *helpers.Helpers
 
-	integrationTest integrationTypes.IntegrationTest
+	integrationTest             integrationTypes.IntegrationTest
+	toastsDuringIntegrationTest chan string
```

Comment:
Is this the best place to store the state?

Reply:
I'd put it in `GuiDriver` next to `isIdleChan`

Reply:
I thought so too at first, but then couldn't find a way how to get at it from the production code. I may have overlooked something obvious, will try again.

Reply:
The only way I could find to make this work is to add a `SetToastFunc` method to `PopupHandler` so that we can set it after construction in `handleTestMode`. I still like this better than before. See c630af095bc4